frontend/src/__tests__/cypress/cypress/e2e/modelRegistry/ModelRegistry.cy.ts

```diff
@@ -14,8 +20,31 @@ const initIntercepts = ({ disableModelRegistryFeature = false }: HandlersProps)
     }),
   );
   cy.interceptOdh('GET /api/components', { query: { installed: 'true' } }, mockComponents());
+
+  cy.interceptK8sList(
+    ModelRegistryModel,
+    mockK8sResourceList([mockModelRegistry({}), mockModelRegistry({ name: 'test-registry' })]),
```

Comment:
When using `cy.interceptK8sList`, I am receiving an empty result from the network call: `{"apiVersion":"unknown","metadata":{},"items":[]}`. 

However, it works fine with `cy.interceptK8s`

@christianvogt, could you please check?

Reply:
@dpanshug k8s requests require the namespace to be part of the URL. Therefore for ease of use `interceptK8sList` will extract the namespace from the first entry in the response list of k8s resources (this can be overridden in the param options but often is unnecessary if the correct resource is supplied). The namespace from the mock looks to be `odh-model-registries`. Please ensure that the frontend is making a request to the correct namespace. It looks to me like that may not be the case as the frontend code does not include a namespace in the request:
```
export const listModelRegistries = async (): Promise<ModelRegistryKind[]> =>
  k8sListResource<ModelRegistryKind>({
    model: ModelRegistryModel,
  }).then((listResource) => listResource.items);
```

Reply:
thanks, got the answer.